### DISCUSSION TITLE
Duplicate (or clone) an item

### DISCUSSION THREAD

**Discussion by jordie23** [keystonejs/keystone#6867] [Feature requests]
Would be great to have a "clone" button for list items.

I'd imagine the button could be located in various places, such as the list index page, or the edit an item page. Clicking the button would open up a new create item panel prefilled with all the values from the existing item. 

Tried searching for this and couldn't see it requested yet so thought I'd pop it in here.

Perhaps the ability to opt-out of this ability in the item schema. `allowCloning: false` or something :)

**Comment by rafalenden** [comment]
I would love that feature!

**Reply by gautamsi** [reply]
you can still do that with a virtual field with position on sidebar
